script/dynflow-sidekiq.rb

```diff
@@ -0,0 +1,25 @@
+rails_root = Dir.pwd
```

Comment:
Does this script need a shebang? Could also drop the file extension then.

Reply:
It's required by the sidekiq, what is ruby process already. So we need right shebang in the sidekiq, not here.

Reply:
Does the sidekiq bin file end up with the right SCL shebang?

Reply:
It's configured in [foreman-packaging](https://github.com/theforeman/foreman-packaging/pull/4194/files#diff-3399e5c03dc282964481f8f3ced9c141R737) (not sure if correctly though)

Reply:
Typically `script` includes executable scripts. The javascript files are exceptions and should really be fixed. If this is not intended as an executable file then maybe `extras` is a better place.